src/server/bitops_family.cc

```diff
@@ -4,21 +4,50 @@
 
 #include "server/bitops_family.h"
 
+#include <algorithm>
+#include <array>
 #include <bitset>
+#include <cstddef>
+#include <cstdint>
+#include <ext/alloc_traits.h>
+#include <iterator>
+#include <limits>
+#include <memory>
+#include <new>
+#include <numeric>
+#include <ostream>
+#include <string>
+#include <string_view>
+#include <system_error>
+#include <type_traits>
+#include <utility>
+#include <vector>
 
 extern "C" {
+#include "absl/numeric/bits.h"
```

Comment:
It seems the tool does not work well with extern c includes. Surprised it compiles this way

Reply:
Following the steps in the PR description the tool makes (very) breaking changes, like the one you mentioned. Still working on if I can mess with any commandline or cake args to prevent stuff like that. Otherwise you're right this definitely doesn't compile.